WordPress/src/main/res/layout/notifications_detail_activity.xml

```diff
@@ -14,33 +34,17 @@
             android:id="@+id/toolbar_main"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+
             app:theme="@style/WordPress.ActionBar" />
 
     </com.google.android.material.appbar.AppBarLayout>
 
     <org.wordpress.android.widgets.WPViewPager
         android:id="@+id/viewpager"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        app:layout_behavior="@string/appbar_scrolling_view_behavior" />
-
-    <ProgressBar
-        android:id="@+id/progress_loading"
         android:layout_width="wrap_content"
+        android:background="@color/placeholder"
```

Comment:
Changed to placeholder and committed.

Reply:
![Screenshot 2021-08-30 at 1 20 06 PM](https://user-images.githubusercontent.com/54184277/131325897-4faa8fd2-f48d-4e47-8e9c-bf1d9a012833.png)
![Screenshot 2021-08-30 at 1 20 36 PM](https://user-images.githubusercontent.com/54184277/131325914-d69c91b1-42b2-4703-be3e-b99a1ff4c835.png)
Modified it for both Light And Dark Mode.

Reply:
Thank you! Let's wait till we get a design review.

Reply:
As discussed, we're merging this and we'll capture [suggested animation](https://github.com/wordpress-mobile/WordPress-Android/issues/14919#issuecomment-908503715) in a separate task.